src/main/scala/se/nimsa/sbx/box/BoxDAO.scala

```diff
@@ -40,7 +40,7 @@ class BoxDAO(val dbConf: DatabaseConfig[JdbcProfile])(implicit ec: ExecutionCont
 
   class BoxTable(tag: Tag) extends Table[Box](tag, BoxTable.name) {
     def id = column[Long]("id", O.PrimaryKey, O.AutoInc)
-    def name = column[String]("name", O.Length(255))
```

Comment:
I had to reduce the length of indexed columns for compliance with MySql when using utf-8. Each character is then encoded by ~3 bytes and the hard limit on these columns in MySQL is 767 bytes.